app/src/main/java/de/westnordost/streetcomplete/quests/address/AddAddressStreet.kt

```diff
@@ -31,12 +31,11 @@ class AddAddressStreet : OsmElementQuestType<AddressStreetAnswer> {
     override val wikiLink = "Key:addr"
     // In Japan, housenumbers usually have block numbers, not streets
     override val enabledInCountries = AllCountriesExcept("JP")
-
     override val questTypeAchievements = listOf(POSTMAN)
 
     override fun getTitle(tags: Map<String, String>) = R.string.quest_address_street_title
 
-    override fun getTitleArgs(tags: Map<String, String>, featureName: Lazy<String?>): Array<String> =
+    override fun getTitleArgs(tags: Map<String, String>): Array<String> =
         arrayOfNotNull(tags["addr:streetnumber"] ?: tags["addr:housenumber"] ?: tags["addr:housename"])
```

Comment:
Is it deliberate that this question (`What street does (house) […] belong to?`) is not simplified?

Is it because the tag priority here (`addr:streetnumber` > `addr:housenumber` > `addr:housename`) is different from the logic in `Resources.getHouseNumberLabel`?

Reply:
`AddAddressStreet` is asked not only for buildings. So it would be difficult to refer to the "thing" without saying housenumber. Or do you have an idea how to word it?

Reply:
Why not leave it out completely?

> **What street does this belong to?**
> house number 27

Reply:
This what? Are you sure that it is possible even in English to word it like that?

Reply:
I see no problems with that (but I might be wrong of course).

We have basically the same thing with "Is this still here?" and that seems to be a common expression: https://www.google.com/search?q=%22Is+this+still+here%3F%22

Reply:
@1ec5 suggested "What street is this on?", so I'll just use that

Reply:
I'd never got round to opening an issue, but it always used to but me when I was asked `What street does (house) […] belong to?` for a railway station or an office, so this is a great improvement!